### DISCUSSION TITLE
Copier template for bootstrapping a new module

### DISCUSSION THREAD

**Discussion by yajo** [OCA/odoo-community.org#80] [Show and tell]
Hi friends!

We were a bit bored of copying https://github.com/OCA/maintainer-tools/tree/master/template/module and modifying all stuff. And odoo scaffold is not so good...

So, I created https://gitlab.com/moduon/odoo-module-template .

See its readme to know how to bootstrap a new module from scratch. It includes all standards from https://github.com/OCA/maintainer-tools (actually I copied the whole module template from there, with commit history included).

We can donate the template to OCA if you consider it interesting, or otherwise just use it freely. MRs welcome, as usual!

Happy weekend everybody.

---

Cross-posting from https://odoo-community.org/groups/contributors-15/contributors-269338?mode=thread&date_begin=&date_end= just to get a taste of the new forum 😋

**Comment by rousseldenis** [comment]
@yajo FYI, we use this: https://github.com/acsone/bobtemplates.odoo
